book/_config.yml

```diff
@@ -35,7 +35,7 @@ parse:
 # See https://jupyterbook.org/content/execute.html
 execute:
   # NOTE: 'cache' to use jupyter-cache, or 'force' to run all
-  execute_notebooks: 'force'
+  execute_notebooks: 'off'
```

Comment:
just turning off execution, so trusting things work as is on cryocloud jupyterhub. If you want to commit *rendered* notebooks with outputs we can try that in a new PR